vm-bootstrap.sh

```diff
@@ -21,7 +21,19 @@ fi
 
 cd /opt/Espressif/crosstool-NG
 
-function gitsha() { git show --format=%H | head -1; }
+# Git complains if these values are not set
```

Comment:
I thought the same but without it the provisioning was failing. You can test it by getting this file and then running:

vagrant provision